### PR TITLE
Update text on intellisense

### LINKED ISSUE #29186: Remarks and Summary tags with Intellisense
Hello,

Starting with Visual Studio `v16.6.5`, there is an option in `Text Editor > C# > Advanced > Editor Help`, called `Show remarks in Quick Info`. 

So perhaps the sentence _"The text for the summary tag is the only source of information about the type in IntelliSense, and is also displayed in the Object Browser window."_ in the `summary` section needs extra clarification.

Thank you and best regards!
S.L.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3bcb925f-ef72-9a64-3762-c0b4a96f9dc4
* Version Independent ID: d135d0a6-53d2-1567-0b1b-7aae5e07739f
* Content: [Recommended XML documentation tags for a class and its members](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#summary)
* Content Source: [docs/csharp/language-reference/xmldoc/recommended-tags.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/xmldoc/recommended-tags.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

---
[Associated WorkItem - 215487](https://dev.azure.com/msft-skilling/Content/_workitems/edit/215487)

### LINKED ISSUE #32118: Consider adding usage examples for the `<include>` xml element
This is about the `<include>` section specifically: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc

There should be examples of how to structure the target xml file, as well as different ways of using the `file`, and `path` attributes, as well as the `@name` and `id`.

Unfortunately, I could not find any official documentation showing how to use this, and I had to resort to going deep into the google rabbit hole to find some examples from years ago. ☹️

The good news is that I am currently investigating this, so as soon as I have some working examples, I will share them here.

cc @gewarren

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3bcb925f-ef72-9a64-3762-c0b4a96f9dc4
* Version Independent ID: d135d0a6-53d2-1567-0b1b-7aae5e07739f
* Content: [Recommended XML documentation tags for a class and its members](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags)
* Content Source: [docs/csharp/language-reference/xmldoc/recommended-tags.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/xmldoc/recommended-tags.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

---
[Associated WorkItem - 215486](https://dev.azure.com/msft-skilling/Content/_workitems/edit/215486)

### LINKED ISSUE #38985: XML Comment Documentation out of date
### Type of issue

Outdated article

### Description

Under the &lt;summary&gt; tag, it says
> The text for the &lt;summary&gt; tag is the only source of information about the type in IntelliSense, and is also displayed in the Object Browser window.

This does not appear to be true anymore. VS2022 (17.8.3) will happily display all the remarks, examples, etc. too.

### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/xmldoc/recommended-tags.md

### Document Version Independent Id

d135d0a6-53d2-1567-0b1b-7aae5e07739f

### Article author

@BillWagner

### Metadata

* ID: 3bcb925f-ef72-9a64-3762-c0b4a96f9dc4 
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #38985
Fixes #29186

Other tags in addition to `<summary>` may appear on intellisense, depending on the version of VS and the user's settings. Relax some of the statements on that.

In addition, looked for other issues on this article:

Fixes #32118

Add an example showing how to use the include tag.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/csharp/language-reference/xmldoc/recommended-tags.md](https://github.com/dotnet/docs/blob/15fe122ed60424b7cd60c98690e04b205abd7268/docs/csharp/language-reference/xmldoc/recommended-tags.md) | [Recommended XML tags for C# documentation comments](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags?branch=pr-en-us-39642) |

### CODE REVIEW

**Comment by BillWagner** [comment]
ping @adegeo for review

File: docs/csharp/language-reference/xmldoc/recommended-tags.md [hunk 1]
```diff
@@ -319,6 +319,26 @@ Add your XML comments in base classes or interfaces and let inheritdoc copy the
...
+
+```xml
+<member name="M:MyNamespace.MyType.MyMethod(System.Int32)">
+    <returns>This is the returns text of MyMethod. It comes from triple slash comments.</returns>
+    <remarks>This is the remarks text of MyMethod. It comes from triple slash comments.</remarks>
+    <param name="p">This is the description of the parameter p of MyMethod. It comes from the included file.</param><summary>This is the summary of MyMethod. It comes from the included file.</summary>
+</member>
+```
+
+The .NET Runtime team uses the `<include>` tag extensively in its documentation. You can see many examples by [searching the `dotnet/runtime` repository](https://github.com/search?q=repo%3Adotnet%2Fruntime+language%3Acsharp+%3Cinclude+file+&type=code).
```
Comment (gewarren) [inline docs/csharp/language-reference/xmldoc/recommended-tags.md:340]: Should this be a TIP block?

**Review by gewarren — CHANGES_REQUESTED** [review]

File: docs/csharp/language-reference/xmldoc/recommended-tags.md [hunk 2]
```diff
@@ -319,6 +319,26 @@ Add your XML comments in base classes or interfaces and let inheritdoc copy the
...
+
+:::code language="xml" source="./snippets/xmldoc/MyAssembly.xml":::
+
+The XML output for this method is shown in the following example:
+
+```xml
+<member name="M:MyNamespace.MyType.MyMethod(System.Int32)">
+    <returns>This is the returns text of MyMethod. It comes from triple slash comments.</returns>
+    <remarks>This is the remarks text of MyMethod. It comes from triple slash comments.</remarks>
+    <param name="p">This is the description of the parameter p of MyMethod. It comes from the included file.</param><summary>This is the summary of MyMethod. It comes from the included file.</summary>
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/csharp/language-reference/xmldoc/recommended-tags.md:336]: typo. I'll fix.

File: docs/csharp/language-reference/xmldoc/recommended-tags.md (hunk 1 above)
Comment (BillWagner) [inline docs/csharp/language-reference/xmldoc/recommended-tags.md:340]:
```suggestion
> [!TIP]
> The .NET Runtime team uses the `<include>` tag extensively in its documentation. You can see many examples by [searching the `dotnet/runtime` repository](https://github.com/search?q=repo%3Adotnet%2Fruntime+language%3Acsharp+%3Cinclude+file+&type=code).
```

**Comment by BillWagner** [comment]
Thanks @gewarren 

I've addressed all your comments.

**Review by gewarren — APPROVED** [review]

**Pull request merged**